Makefile

```diff
@@ -355,6 +355,7 @@ endif
 		$(GO_LDFLAGS) '$(LDFLAGS_PODMAN)' \
 		-tags "$(BUILDTAGS)" \
 		-o $@ ./cmd/podman
+	test -z "${SELINUXOPT}" || chcon -t container_runtime_exec_t $@
```

Comment:
LGTM but I really, really hate `||` being used this way. I would super-strongly prefer:
```bash
    test -n ... && chcon
```
...which makes it clear that it's an `if-then`. `||` should IMO be reserved for uh-oh statements:
```bash
    test -z "" ... || warn "Yikes, found something wrong".
```
My guideline is, `||` points the eye at the left-hand side of the statement being the important action, `&&` points to the right.

There's also a teeny tiny nit, the distinction between "SELinux is enabled" (`selinuxenabled`) and "SELinux is *available*", my preference being to `chcon` in the latter case, but I can't think of any real-world cases where this matters.

I won't block for either. LGTM, and thank you! I've had to train myself to `hack/bats sdfsdfsdf` after `make`, simply to get the chcon. This is very helpful.

Reply:
> There's also a teeny tiny nit, the distinction between "SELinux is enabled" (selinuxenabled) and "SELinux is available", my preference being to chcon in the latter case, but I can't think of any real-world cases where this matters.

I don't understand what you are trying to say with this? Are saying we should label even if selinux is disabled?
Note that selinuxenabled even return true in the case where `setenforce 0` (permissive), it only returns false if is is completely disabled at boot time AFAICT.

Reply:
>selinuxenabled even return true in the case where setenforce 0

I did not realize that. It is not exactly clear from the man page. Thank you for setting me straight, I withdraw my second nit.

/lgtm